### ISSUE TITLE
Focus on all colored buttons relies solely on subtle color change

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1140]
"Description: When a user navigates to a colored button (e.g. ‘Configure Theme’, ‘Reset Navigation’, ‘Save Changes’, ‘Use this theme’, etc.) using keyboard navigation, the only visual indication that the button is focused is a slight darkening of the button’s background. There is no outline, border, or other shape of change. Users with color vision deficiencies or low vision may not perceive this subtle color change, making it difficult to identify the focused element. 

Note: This also occurs on the,  
- 'Admin Dashboard' page with the 'Add a new item' and 'Add a new Collection' buttons
- 'Appearance' Theme page with the 'Configure Theme' and 'Use this theme' buttons
- 'Appearance' Navigation page with the 'Save Changes' and 'Reset Navigation' buttons
- 'Plugins' page with the 'Activate', 'Uninstall', 'Configure', 'Deactivate', and 'Install' buttons 
- 'Configure Plugin' page with the 'Save Changes' button
- 'Contribution' page in the 'Contributor Questions' sections for the "Save Changes" and "Delete" button
- Configure Plugin: Simple Contact Form Toolbar More Options Insert/Edit Link and Source Code Modal Cancel/Save Buttons

Expected Results: Keyboard focus should be clearly indicated using more than just a color change, such as an outline, border, or other visual indicator, so that all users, including those with color vision deficiencies, can perceive which element is focused. 

Actual Results: The only visual cue for focus is a minor darkening of the button background, which may not be perceivable to all users."

1.4.1 Use of Color; Level A; WCAG 2.0

**Comment by kimisgold** [comment]
This should be a fairly simple fix, just removing all the `outline: none` styles from the buttons. The changes are on the `master` branch.

**Comment by aviannamiller** [comment]
Great, that looks to cover it.

Any way to tighten up the bottom of the outlines on these browse pagination arrow buttons? Not a huge deal if not:
<img width="259" height="105" alt="Pagination controls. Focus is on previous arrow button, and the outline extends below the bottom of the button." src="https://github.com/user-attachments/assets/b1ade14b-a340-4eb5-b679-59cf64a7a4d3" />

Probably unrelated to this issue, but I noticed the Configure buttons on plugins have less height, on mobile/narrow views only. I don't think this has any connection to these changes but worth mentioning all the same:

<img width="534" height="278" alt="PDF Text plugin listing with buttons to configure, deactivate, and uninstall. The configure button is shorter than the rest." src="https://github.com/user-attachments/assets/772b1445-ab59-4daa-9a25-5f779923edf2" />

 Either way, they're still in an appropriate range for touch target sizing minimum guidelines.

**Comment by kimisgold** [comment]
Both issues should be addressed now. The pagination link had some styles that were making the pseudo element larger than the container, and the configure buttons were getting some overreaching styles targeting action links on the browse view tables.

**Comment by aviannamiller** [comment]
Looks great, thank you!
